frontend/app/model/Identity.scala

```diff
@@ -56,3 +58,36 @@ object AccessCredentials {
 }
 case class IdMinimalUser(id: String, displayName: Option[String])
 case class AuthenticatedIdUser(credentials: AccessCredentials, minimalUser: IdMinimalUser)
+
+// identity-play-auth (which previously provided CookieBuilder) has been deprecated in favour of identity-auth-play:
```

Comment:
🙈 couldn't it have been called identity-play-auth2 then at least it's clear which is the new one?  with an arbitrary change it doesn't really help in the short term.

Reply:
yep, fair point :/